index.html

```diff
@@ -24,45 +29,97 @@ <h1 class="blogName"><img src ="images/logo.png" id="logo">
+          <p/>
+          <p class="instructions">
+            Ice cream: In a small pot over medium heat, combine milk, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled.<br/>
+          </p>
+          <p class="instructions">
+            Using an ice cream machine, pour liquids into the frozen basin and process according to manufacturer instructions, i.e. let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.<br/>
+          </p>
+        </section>
+        <section class="ingredientsList">
+          <table>
+            <tr>
+              <td>
+                <input type="checkbox" name="ingredient" value="milk"/>1 1/2 cups milk
+              </td>
+              <td>
+                <input type="checkbox" name="ingredient" value="mascarpone"/>1 1/2 cups mascarpone
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <input type="checkbox" name="ingredient" value="pinkSalt"/>1/2 tsp pink salt
+              </td>
+              <td>
+                <input type="checkbox" name="ingredient" value="figs"/>1lb Black Mission Figs
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <input type="checkbox" name="ingredient" value="brownSugar"/>1/2 cup brown sugar
+              </td>
+              <td>
+                <input type="checkbox" name="ingredient" value="water"/>2-4 tbsp water
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <input type="checkbox" name="ingredient" value="cream"/>1 1/2 cups heavy cream
+              </td>
+              <td>
+                <input type="checkbox" name="ingredient" value="granSugar" checked="checked"/>1/3 granulated sugar
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <input type="checkbox" name="ingredient" value="eggs" checked="checked"/>2 egg yolks
+              </td>
+              <td>
+                <input type="checkbox" name="ingredient" value="lemon"/>1 lemon, juiced
+              </td>
+            <tr>
+              <td>
+                <input type="checkbox" name="ingredient" value="butter"/>2-4 tbsp butter
+              </td>
+              <td>
+                <input type="checkbox" name="ingredient" value="pecans"/>1 cup honey roasted pecans, roughly<br/>chopped
+              </td>
+            </tr>
+          </table>
+        </section>
+      </article>
+      <footer>
```

Comment:
I would argue that the author info belongs to the article and not the footer. If we think of the footer in a dynamic way, it's a module that seldom changes and stays consistent throughout pages, whereas the article is prone to change depending on the article.